src/core/lib/surface/completion_queue.cc

```diff
@@ -411,12 +411,13 @@ static const cq_vtable g_cq_vtable[] = {
 
 grpc_core::TraceFlag grpc_cq_pluck_trace(false, "queue_pluck");
 
-#define GRPC_SURFACE_TRACE_RETURNED_EVENT(cq, event)                       \
-  if (grpc_api_trace.enabled() && (grpc_cq_pluck_trace.enabled() ||        \
-                                   (event)->type != GRPC_QUEUE_TIMEOUT)) { \
-    char* _ev = grpc_event_string(event);                                  \
-    gpr_log(GPR_INFO, "RETURN_EVENT[%p]: %s", cq, _ev);                    \
-    gpr_free(_ev);                                                         \
+#define GRPC_SURFACE_TRACE_RETURNED_EVENT(cq, event)    \
+  if (GRPC_TRACE_FLAG_ENABLED(grpc_api_trace) &&        \
+      (GRPC_TRACE_FLAG_ENABLED(grpc_cq_pluck_trace) ||  \
+       (event)->type != GRPC_QUEUE_TIMEOUT)) {          \
+    char* _ev = grpc_event_string(event);               \
+    gpr_log(GPR_INFO, "RETURN_EVENT[%p]: %s", cq, _ev); \
+    gpr_free(_ev);                                      \
```

Comment:
... and likewise with this?

Reply:
Thank you for noting this. Sure converted this and 4 others over.